src/main/scala/com/github/takezoe/slick/blocking/BlockingProfile.scala

```diff
@@ -56,8 +56,8 @@ trait BlockingJdbcProfile extends BlockingRelationalProfile { profile: JdbcProfi
       }
     }
   
-    implicit class RepQueryExecutor[E: BaseTypedType](rep: Rep[E]){
```

Comment:
reverting the commit that introduced this regression wasn't a solution because compilation in scala 2.10 was still failing.  the solution here is to directly call slick.lifted.RepShape.apply which is what other implicit conversions were doing under the wood